README.md

```diff
@@ -67,6 +67,15 @@ See the following eff.org blog post for more information: [Giving Privacy Badger
    $ ./runscan.sh --exclude org,net
    ```
 
+   You can load another extension to run in parallel to Privacy Badger during a scan.
+   Use the `--load-extension` flag and pass along the filepath for the unpacked extension
```

Comment:
Let's load packed everywhere for consistency. CRXs in Chrome and zips in Firefox.